e2e/test/E2ETests.csproj

```diff
@@ -24,6 +24,7 @@
     <PackageReference Include="MSTest.TestFramework" Version="2.0.0" />
     <PackageReference Include="Microsoft.CSharp" Version="4.7" />
     <PackageReference Include="System.Diagnostics.Tracing" Version="4.3.0" />
+    <PackageReference Include="WindowsAzure.Storage" Version="9.3.2" />
```

Comment:
We should use the latest SDK, not this old and deprecated one. That might be another work item and PR, but we should track it.

Look for `Azure.Storage.*` packages.